app/scripts/dimApp.i18n.js

```diff
@@ -121,16 +121,16 @@
           Junk: "Trödel",
           Infuse: "Infundieren",
           notes_placeholder: "Hinzufügen von Anmerkungen zu diesem Artikel",
-          'Create Loadout': "Erstellen Ausladen",
```

Comment:
Yeah I agree with @Korben85, and this is my fault as I found a translation for Loadout, let's just leave loadout as loadout, so Erstellen Loadout, etc